src/ng/plugins/groupbar/groupbar.js

```diff
@@ -15,7 +15,7 @@ class Groupbar extends PluginComponent('qgrid.groupbar.tpl.html') {
 						by: state.by.concat(key)
 					});
 
-					this.newGroup = null;
+					this.newSort = null;
```

Comment:
Revert name from newSort to newGroup (for src/ng/plugins/groupbar/groupbar.js file)